torch/_inductor/lowering.py

```diff
@@ -2190,7 +2192,7 @@ def new_constant(fill_value):
     def _new_constant(
         x, size, *, dtype=None, layout=None, device=None, pin_memory=None
     ):
-        assert isinstance(size, (list, type))
+        assert isinstance(size, (list, tuple))
```

Comment:
do we not want the type here ? worried this might be semantics chaging

Reply:
Yeah, well, the downstream code doesn't check anything and does things like `len(size)`, which (unless the `type` is some very unusual type) would break. So I thought this was supposed to be a tuple. Can change back and add `# type ignore` if you think it'd be better.